### PR TITLE
Refactor billing

### PR DESCRIPTION
**Pull request by murrant**
Prevents issues of usages where the code hasn't been loaded.

#### Please note

> Please read this information carefully. You can run `./lnms dev:check` to check your code before submitting.

- [x] Have you followed our [code guidelines?](https://docs.librenms.org/Developing/Code-Guidelines/)
- [x] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.
- [x] If my Pull Request makes discovery/polling/yaml changes, I have added/updated [test data](https://docs.librenms.org/Developing/os/Test-Units/).

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

File: includes/html/graphs/bill/historicmonthly.inc.php [hunk 1]
```diff
@@ -4,9 +4,10 @@
 use Amenadiel\JpGraph\Plot\BarPlot;
 use Amenadiel\JpGraph\Plot\GroupBarPlot;
 use Amenadiel\JpGraph\Plot\LinePlot;
+use LibreNMS\Billing;
 use LibreNMS\Util\Number;
 
-$graph_data = getHistoricTransferGraphData($vars['id']);
+Billing::getHistoricTransferGraphData($vars['id']);
```
Comment (Jellyfrog) [inline includes/html/graphs/bill/historicmonthly.inc.php:10]: Missing variable assignment?
Probably causing https://github.com/librenms/librenms/issues/14994

Comment (murrant) [reply includes/html/graphs/bill/historicmonthly.inc.php:10]: Hmm, I checked all the graphs, thanks.

File: includes/html/api_functions.inc.php [hunk 2]
```diff
@@ -1634,9 +1635,9 @@ function get_bill_graphdata(Illuminate\Http\Request $request)
             $to = $request->get('to', time());
             $reducefactor = $request->get('reducefactor');
 
-            $graph_data = getBillingBitsGraphData($bill_id, $from, $to, $reducefactor);
+            $graph_data = Billing::getBitsGraphData($bill_id, $from, $to, $reducefactor);
         } elseif ($graph_type == 'monthly') {
-            $graph_data = getHistoricTransferGraphData($bill_id);
+            Billing::getHistoricTransferGraphData($bill_id);
```
Comment (Jellyfrog) [inline includes/html/api_functions.inc.php:1640]: Same here?

File: includes/html/graphs/bill/historictransfer.inc.php [hunk 3]
```diff
@@ -4,14 +4,15 @@
...
 
 if (is_numeric($vars['bill_hist_id'])) {
-    $graph_data = getBillingBandwidthGraphData($vars['id'], $vars['bill_hist_id'], null, null, $vars['imgtype']);
+    $graph_data = Billing::getBandwidthGraphData($vars['id'], $vars['bill_hist_id'], null, null, $vars['imgtype']);
 } elseif (is_numeric($vars['from'])) {
-    $graph_data = getBillingBandwidthGraphData($vars['id'], null, $vars['from'], $vars['to'], $vars['imgtype']);
+    $graph_data = Billing::getBandwidthGraphData($vars['id'], null, $vars['from'], $vars['to'], $vars['imgtype']);
 } else {
-    $graph_data = getHistoricTransferGraphData($vars['id']);
+    Billing::getHistoricTransferGraphData($vars['id']);
```
Comment (Jellyfrog) [inline includes/html/graphs/bill/historictransfer.inc.php:15]: Same here?

Comment (murrant) [reply includes/html/graphs/bill/historictransfer.inc.php:15]: omg

Comment (murrant) [reply includes/html/graphs/bill/historictransfer.inc.php:15]: Must have been the stupid refactoring tool that messed up.

**Pull request merged**